<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StructureOuTexteEdit</class>
 <widget class="QWidget" name="StructureOuTexteEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>36</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="StructureEdit" name="structureEdit" native="true"/>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonBasculer">
     <property name="text">
      <string>Basculer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StructureEdit</class>
   <extends>QWidget</extends>
   <header>vue/structureedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
